lfads/plot_lfads.py

```diff
@@ -182,8 +181,6 @@ def plot_lfads_timeseries(data_bxtxn, model_vals, ext_input_bxtxi=None,
     means_nxt = params_nxt
   elif output_dist == 'gaussian': # (means+vars) x time
     means_nxt = np.vsplit(params_nxt,2)[0] # get means
-  else:
-    assert "NIY"
```

Comment:
instead of deleting entirely, want:
`raise ValueError("output_dist '{}' not implemented.".format(output_dist))`